.github/workflows/ci.yml

```
@@ -469,58 +576,46 @@ jobs:
           message: Cut a release for a new Dart Sass version
           commit: --allow-empty
 
-  release_embedded_compiler:
-    name: "Release Embedded Compiler"
+  release_embedded_host:
+    name: "Release Embedded Host"
     runs-on: ubuntu-latest
-    needs: [deploy_pub, deploy_sub_packages]
-    if: "startsWith(github.ref, 'refs/tags/') && github.repository == 'sass/dart-sass'"
-
+    needs: [deploy_github_linux, deploy_github_linux_qemu, deploy_github]
+    if: "startsWith(github.ref, 'refs/tags/') && github.repository == 'sass/dart-sass-embedded'"
     steps:
       - uses: actions/checkout@v3
         with:
-          repository: sass/dart-sass-embedded
+          repository: sass/embedded-host-node
           token: ${{ secrets.GH_TOKEN }}
-      - uses: dart-lang/setup-dart@v1
-      - uses: frenck/action-setup-yq@v1
-        with: {version: v4.30.5} # frenck/action-setup-yq#35
 
       - name: Get version
         id: version
-        run: |
-          echo "sass=${GITHUB_REF##*/}" | tee --append $GITHUB_OUTPUT
-          echo "sass_api=$(curl --fail --silent --show-error --location https://raw.githubusercontent.com/sass/dart-sass/${GITHUB_REF##*/}/pkg/sass_api/pubspec.yaml | yq .version)" | tee --append $GITHUB_OUTPUT
+        run: echo "::set-output name=version::${GITHUB_REF##*/}"
```

```suggestion
        run: echo "version=${GITHUB_REF##*/}" | tee --append $GITHUB_OUTPUT
```

`::set-output` is deprecated:

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

---

Looks like we may have a few of these. Maybe we should do this separately in a different PR.

---

Yeah, I'd prefer to save this for a separate PR.